docs/source/markdown/options/uidmap.container.md

```diff
@@ -76,8 +84,112 @@ Every additional range is added sequentially afterward:
+the rest of subordinate ids to be mapped by podman at will.
+
+  `Passing only one of --uidmap or --gidmap`
+
+Usually, subordinated user and group ids are assigned simultaneously, and
+for any user the subordinated user ids match the subordinated group ids.
+For convenience, if only one of **--uidmap** or **--gidmap** is given,
+podman assumes the mapping refers to both UIDs and GIDs and applies the
+given mapping to both. If only one value of the two needs to be changed,
+the mappings should include the `u` or the `g` flags to specify that
+they only apply to UIDs or GIDs and should not be copied over.
+
+ flag    | Example         | Description
+---------|-----------------|-----------------
+  `u`    | `u20000:2000:1` |The mapping only applies to UIDs
+  `g`    | `g10000:1000:1` |The mapping only applies to GIDs
+
+For instance given the command
+
+    podman <<subcommand>> --gidmap "0:0:1000" --gidmap "g2000:2000:1"
+
+Since no **--uidmap** is given, the **--gidmap** is copied to **--uidmap**,
+giving a command equivalent to
+
+    podman <<subcommand>> --gidmap "0:0:1000" --gidmap "2000:2000:1" --uidmap "0:0:1000"
+
+The `--gidmap "g2000:2000:1"` used the `g` flag and therefore it was
+not copied to **--uidmap**.
+
+  `Rootless mapping of additional host GIDs`
+
+A rootless user may desire to map a specific host group that has already been
+subordinated within _/etc/subgid_ without specifying the rest of the mapping.
+
+This can be done with **--gidmap "+g*container_gid*:@*host_gid*"**
+
+Where:
+
+- The host GID is given through the `@` symbol
+- The mapping of this GID is not copied over to **--usermap** thanks to the `g` flag.
+- The rest of the container IDs will be mapped starting from 0 to n,
+  with all the remaining subordinated GIDs, thanks to the `+` flag.
+
+For instance, if a user belongs to the group `2000` and that group is
+subordinated to that user (with `usermod --add-subgids 2000-2000 $USER`),
+the user can map the group into the container with: **--gidmap=+g100000:@2000**.
+
+If this mapping is combined with the option, **--group-add=keep-groups**, the
+process in the container will belong to group `100000`, and files belonging
+to group `2000` in the host will appear as being owned by group `100000`
+inside the container.
+
+    podman run --group-add=keep-groups --gidmap="+100000:@2000" ...
+
+  `No subordinate UIDs`
+
 Even if a user does not have any subordinate UIDs in  _/etc/subuid_,
 **--uidmap** can be used to map the normal UID of the user to a
 container UID by running `podman <<subcommand>> --uidmap $container_uid:0:1 --user $container_uid ...`.
 
-Note: the **--uidmap** flag cannot be called in conjunction with the **--pod** flag as a uidmap cannot be set on the container level when in a pod.
+  `Pods`
+
+The **--uidmap** flag cannot be called in conjunction with the **--pod** flag as a uidmap cannot be set on the container level when in a pod.
```

Comment:
Since you are introducing the use of "flags", it's even more important to call the options "options" rather than flags.
```suggestion
The **--uidmap** option cannot be called in conjunction with the **--pod** option as a uidmap cannot be set on the container level when in a pod.
```